mace/modules/models.py

```diff
@@ -254,120 +187,364 @@ def forward(
+        correlation
+        gate
+            non-linearity for non-linear readouts
+        radial_MLP
+
+        # ------- energy model specific parameters
+        atomic_energies
+            energy shift (e0) of elements
+            order & length to agree with atomic_numbers
+        atomic_inter_scale
+            scale of interaction energy
+        atomic_inter_shift
+            constant shift of interaction energy (per atom)
+
+        **kwargs
+        """
         super().__init__(**kwargs)
-        self.scale_shift = ScaleShiftBlock(
-            scale=atomic_inter_scale, shift=atomic_inter_shift
-        )
+
+
+@compile_mode("script")
+class AtomicDipolesMACE(MaceCoreModel, DipoleModelMixin):
+    """MACE model for dipoles only"""
+
+    _LINEAR_READOUT_CLASS = LinearDipoleOnlyReadoutBlock
+    _NONLINEAR_READOUT_CLASS = NonLinearDipoleOnlyReadoutBlock
+
+    def __init__(self, hidden_irreps: o3.Irreps, atomic_energies=None, **kwargs):
+        """MACE dipole only model
+
+        Parameters
+        ----------
+        # ------- core model parameters
+        r_max
+            cutoff of radial embedding applied to individual atoms
+        num_bessel
+            number of Bessel functions to be used for radial embedding
+        num_polynomial_cutoff
+        max_ell
+            l_max, maximum channels for spherical harmonics
+        interaction_cls
+            class to be used for interactions blocks
+        interaction_cls_first
+            class to be used for the first layer's interaction block
+        num_interactions
+            number of interaction layers to use
+        num_elements
+            redundant parameter for the number of elements
+        hidden_irreps
+            hidden irreducible representations, basically the size of the layer features
+            and hence direct control on the size of the model
+        MLP_irreps
+        avg_num_neighbors
+        atomic_numbers
+            atomic numbers of elements the model supports
+            order & length to agree with atomic_energies
+        correlation
+        gate
+            non-linearity for non-linear readouts
+        radial_MLP
+
+        **kwargs
+
+        Notes
+        -----
+        Requires at least l=1 features to be used for representing dipoles.
+        `atomic_energies` parameter added for compatibility, but it's ignored
+
+        """
+        assert (
+            hidden_irreps.lmax >= 1
+        ), "To predict dipoles use at least l=1 hidden_irreps"
+        super().__init__(hidden_irreps=hidden_irreps, **kwargs)
+
+    @staticmethod
+    def _last_layer_irreps(hidden_irreps) -> o3.Irreps:
+        """dipole: Select only l=1 vectors for last layer"""
+        return o3.Irreps(str(hidden_irreps[1]))
 
     def forward(
         self,
         data: Dict[str, torch.Tensor],
         training: bool = False,
-        compute_force: bool = True,
+        compute_force: bool = False,
```

Comment:
If this class cannot calculate forces, why don't we remove the associated parameters?  Are they intended for compatibility with wrappers?